creatign and verefiing the cllient (this migt be a change to the ordering system that is implemented in the websites)

ahmed-oues opened the pull request:
(this migt be a change to the ordering system that is implemented in the websites, this pull request might need to wait  )
-client collection created
-order collection updatetd
-added client id and populated as client datat
-client mamangment for veryfiing or banning a client is added

ahmed-oues committed:
creatign and verefiing the cllient

-client collection created
-order collection updatetd
-added client id and populated as client datat
-client mamangment for veryfiing or banning a client is added

Jamel-miraoui commented on .env:
```diff
@@ -11,6 +11,8 @@ MAIL_SENDER = affigeninc@gmail.com
 # MAIL_RECEIVERS = tunisia@gentaur.com info@affigen.com
```
This repository is no longer supported please start using GitLab in the future thank you

ahmed-oues reviewed (commented):
good

ahmed-oues committed:
update on never accseptin orders from uverified email

ahmed-oues committed:
need validation and test befor deploy this

adding the count and analytics to a number of collections
aalong side the part wher we dont allow clients to mke orders if the email is not verified
NB: migt brake the website if not handeled correcly

ahmed-oues reviewed (commented):
needs testing

ahmed-oues committed:
orders + analitcs +hot products

ahmed-oues committed:
separating gentaur hot from affigen hot

ahmed-oues committed:
adding bloges
